spec/forms/hyrax/forms/permission_template_form_spec.rb

```diff
@@ -195,12 +195,14 @@
 
     context "when a new workflow has been chosen" do
       it "gives the managers workflow roles" do
+        expect(form).not_to receive(:update_permission_template)
```

Comment:
Yes, but what I meant by that was there a set of inputs we could use to trigger a specific bug, that is fixed by our changes to the code?  Not "does it follow a specific code path".